README.md

```diff
@@ -52,6 +52,10 @@ optionally (though the build-system might currently require these libraries):
 - libgbm-dev
 - libclang-dev
 - libpipewire-0.3-dev
+- librust-atk-dev
```

Comment:
This is just a Rust crate. The actual dev library is `libatk1.0-dev`.

Reply:
Fixed in https://github.com/pop-os/cosmic-epoch/commit/5008be26589eff882c3ca2441ba34b9a2edd9693